pkg/stub/handler.go

```diff
@@ -629,7 +679,8 @@ func (h *Handler) Handle(event util.Event) error {
 		}
 
 		//TODO adjust this check as we start getting samples for z or ppc
-		if util.IsNonX86Arch(cfg) {
+		// and registry.redhat.io starts support IPv6
+		if util.IsNonX86Arch(cfg) || util.IPv6Only() {
```

Comment:
Similar - if we're removed, do we reach this code path?

Reply:
the code here was more of a paranoid make sure we don't try to install

I think we can remove it at this point.